test/module/evaluators.js

```diff
@@ -103,19 +103,20 @@ test('createSafeEvaluatorWhichTakesEndowments - options.sloppyGlobals', t => {
 
     // Evaluate normally.
     t.equal(safeEval('abc', { abc: 123 }), 123, 'endowment eval');
-    t.throws(
-      () => safeEval('def', { abc: 123 }),
-      ReferenceError,
-      'no such sloppy global'
-    );
-    t.assert(!('def' in sloppyGlobals), 'sloppy global does not yet exist');
+    t.equal(safeEval('typeof def', { abc: 123 }), 'undefined', 'typeof works');
+
+    // FIXME: We can't have both typeof work and a reference error if no such global.
+    t.equal(safeEval('def', { abc: 123 }), undefined, 'no such sloppy global');
+
+    t.assert(!('def' in safeGlobal), 'sloppy global does not yet exist');
```

Comment:
The string `'sloppy global does not yet exist'` is stale.

Check all other mentions of sloppy global for similar staleness.